roles/migrationcontroller/templates/migration-controller.yml.j2

```diff
@@ -0,0 +1,7 @@
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: "{{ meta.name }}"
+  namespace: "{{ meta.namespace }}"
+spec:
+  version: "1.0 (Internal)"
```

Comment:
I am open to suggestions. I want to set something that will make people realize this isn't what they think it is. Doing it this way will just patch the resource after it's created so there's less confusion during the initial creation as well.

Reply:
I feel like adding `(Internal)` might imply that the user has stalled some sort of internal release not intended for distribution. 

If we are going to include a version here, I think it should be meaningful, not just a hard-coded value. Do we have the running version of mig-operator available in an ansible var somewhere we could pull from? Still unclear to me what version means here for OLM.

Reply:
Thoughts on writing the value from `mig_controller_version` ansible var here?